src/test/app/AMMWithdraw_test.cpp

```diff
@@ -0,0 +1,183 @@
+//------------------------------------------------------------------------------
```

Comment:
It can. The reason I wrote it separately was to reduce my iteration time. This let me select just my new tests because they have a new unique name (`ripple.app.AMMWithdraw`). I can fold them in once we're done editing them.